src/Symfony/Component/Serializer/Exception/AggregableExceptionInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Exception;
+
+interface AggregableExceptionInterface
```

Comment:
You are missing an interface here:
```suggestion
interface AggregableExceptionInterface extends ExceptionInterface
```

Reply:
Done